bin/offline-dbg

```diff
@@ -133,6 +142,138 @@ EOF
     echo "**************************************************************"
 }
 
+get_highest_ofp_version() {
+    # if no protocol listed, default to OpenFlow14
+    hex_ofp_version=$(cat $cmd_dir/ovs-ofctl_-t_5_--version | grep "OpenFlow versions" | grep -oP [0-9]*x[0-9]* | tail -1)
```

Comment:
use "{}" in variables at least if they are inside paths
use capitals for global variables or declare with "local"

Reply:
Understood